### ISSUE TITLE
Slow API calls

### ISSUE THREAD

**Issue by matrix07012** [misskey-dev/misskey#5737]
## 💡 Summary

Most interactions cause 100% CPU usage for several seconds and print error into log [api] SLOW API CALL DETECTED

## 📝 Steps to Reproduce

I'm sorry, but there are no steps to reproduce. This issue appeared out of nowhere and has been present for quite a while and I never got around to reporting it.

## 📌 Environment

Ubuntu 18.04.3 LTS (GNU/Linux 4.15.0-74-generic x86_64)
Misskey 11.37.1
Node v12.14.0
Redis 4.0.9
PostgreSQL 10.10

**Comment by mei23** [comment]
When a new user (with no followings) sees Home TL ?

**Comment by mei23** [comment]
Related: 
When someone sees the mention / direct / hashtag TL
When someone sees a photo on the top page
=> #5513

**Comment by matrix07012** [comment]
@mei23 
No, most interactions cause it, but I'm unable to find a pattern. 
For example: Opening one profile might take 2-4s with 100% CPU usage, than opening a different profile takes normal amount of time.
Drive interactions are cause problems often. Opening my drive takes 3-5s and uses 100% of the CPU. Upload from URL often times out due to high CPU usage.
Tags take long time too.

**Comment by cliqsocial** [comment]
@mei23 we are getting the same error messages

![misskey-slow-api-call-detected](https://user-images.githubusercontent.com/60059665/73156428-c4808900-4118-11ea-9e81-5d4b8847da00.png)

**Comment by mei23** [comment]
These messages are diagnostic messages, If there is no impact, you can probably ignore it.
I think there is no problem especially in a few seconds.

**Comment by Casuallynoted** [comment]
I'm experiencing something similar to this, except after taking a long time to load, it fails at the "Something happened :(" prompt
